### PR TITLE
Added Some Sorting & Searching Algorithm for Java

### PR DESCRIPTION
**Pull request by sayeedajmal**

### CODE REVIEW

File: 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
+import java.util.Scanner;
```
Comment (AkashSingh3031) [inline 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java:1]: check this code agaun

Comment (AkashSingh3031) [inline 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java:1]: Check this code

Sorted List not print

File: 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
```
Comment (AkashSingh3031) [inline 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java:1]: Check this code

Sorted List not print

**Review by AkashSingh3031 — CHANGES_REQUESTED** [review]
Sorted List continue to take input 
Check sorting algo

File: 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java (hunk 2 above)
Comment (sayeedajmal) [reply 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java:1]: But Sir.. It's Working....
Screenshot_2021-06-21_19_45_03
**It's Giving Output..**

File: 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java (hunk 1 above)
Comment (sayeedajmal) [reply 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/MergeSort.java:1]: _It's Working..._
It Prints....
Screenshot_2021-06-21_19_49_29

**Comment by sayeedajmal** [comment]
> Sorted List continue to take input
> Check sorting algo

Sorted List with the number of elements which hold how many elements to be entered to be Sorted....

File: 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java (hunk 2 above)
Comment (AkashSingh3031) [reply 1]. DSA/2]. Algorithms/02]. Sorting Algorithms/Java/InsertionSort.java:1]: > But Sir.. It's Working....
> Screenshot_2021-06-21_19_45_03
> **It's Giving Output..**

Ok, I run on an online compiler
So,  problem raise on online compiler

**Review by AkashSingh3031 — COMMENTED** [review]
Accepted

**Pull request merged**